### PR TITLE
forbid space in forcescheduler names

### PR DESCRIPTION
**Pull request by tardyp**
http://trac.buildbot.net/ticket/3460

### CODE REVIEW

File: master/buildbot/schedulers/forcesched.py [hunk 1]
```diff
@@ -616,6 +617,10 @@ def __init__(self, name, builderNames,
             config.error("ForceScheduler name must not be empty: %r " %
                          name)
 
+        if not identifiers.ident_re.match(name):
+            config.error("ForceScheduler name must be an identifier: %r " %
```
Comment (rutsky) [inline master/buildbot/schedulers/forcesched.py]: Is trailing whitespace in the string is necessary?

Can you fix string above too, please?

**Comment by djmitche** [comment]
Looks good otherwise.

**Comment by tardyp** [comment]
updated

**Comment by rutsky** [comment]
LGTM

**Pull request merged**